src/core/public/chrome/ui/header/header_breadcrumbs.tsx

```diff
@@ -85,14 +84,13 @@ export function HeaderBreadcrumbs({
   }));
 
   const remainingCrumbs = useUpdatedHeader ? crumbs.slice(0, -1) : crumbs;
-  const className = useUpdatedHeader ? 'headerBreadcrumbs' : '';
 
   return (
     <EuiHeaderBreadcrumbs
       breadcrumbs={renderFullLength ? crumbs : remainingCrumbs}
       max={10}
       data-test-subj="breadcrumbs"
-      className={className}
+      simplify={useUpdatedHeader ? true : false}
```

Comment:
nit: if `useUpdatedHeader` is a boolean, could just use it directly or `!!useUpdatedHeader`

Reply:
will do